addons/osio-addon/pom.xml

```diff
@@ -15,6 +15,10 @@
       <groupId>io.fabric8.launcher</groupId>
       <artifactId>launcher-core-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.fabric8.launcher</groupId>
```

Comment:
Depend on impl? Better move the Githelper class to API (or the constant to GitService) instead

Reply:
Ok my bad I thought it was already a a transitive dep, but it was in scope test. I will move it ;)
Go in Holiday @gastaldi  loooool